lib/utilityservice.js

```diff
@@ -853,90 +896,14 @@ UtilityService.prototype.getVlocitySetting = async function(settingName) {
 }
 
 UtilityService.prototype.getSfdxCommandRunner = function(jobInfo) {
-    if(jobInfo.useSfdxCli === true && commandExistsSync('sfdx')) {
+    if(!commandExistsSync('sf')){
+        VlocityUtils.log(' Cannot  proceed further Sf command doesnot exist , please install sf');
```

Comment:
Please install SFDX as tool can't proceed further without sf commands.

This should be VlocityUtils.error  or something more appropriate.

Reply:
This change has been accommodated